src/libraries/Fuzzing/DotnetFuzzing/Fuzzers/Base64Fuzzer.cs

```diff
@@ -0,0 +1,105 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+
+using System.Buffers;
+using System.Buffers.Text;
+
+namespace DotnetFuzzing.Fuzzers
+{
+    internal class Base64Fuzzer : IFuzzer
+    {
+        public string[] TargetAssemblies => [];
+
+        public string[] TargetCoreLibPrefixes => ["System.Buffers.Text"];
+
+        public void FuzzTarget(ReadOnlySpan<byte> bytes)
+        {
+            using var inputPoisoned = PooledBoundedMemory<byte>.Rent(bytes, PoisonPagePlacement.After);
```

Comment:
Inappropriate use of `var`

Reply:
All of the uses of var that I see in this file so far are inappropriate.

Reply:
Just followed existing fuzzers style, anyway, will change it https://github.com/dotnet/runtime/blob/c286a8efb3c0d534557dde476d5b54b15c0aeaed/src/libraries/Fuzzing/DotnetFuzzing/Fuzzers/UTF8Fuzzer.cs#L34-L35